src/Algorand/Signer.cpp

```diff
@@ -44,3 +65,19 @@ Data Signer::sign(const PrivateKey &privateKey, Transaction &transaction) noexce
     auto signature = privateKey.sign(data, TWCurveED25519);
     return Data(signature.begin(), signature.end());
 }
+
+Data Signer::sign(const PrivateKey &privateKey, AssetTransaction &transaction) noexcept {
```

Comment:
these three `sign` methods might be unified as one: 
```suggestion
Data Signer::sign(const PrivateKey &privateKey, BaseTransaction transaction) noexcept {
```